tests/slow/src/test/scala/tests/mill/MillLspSuite.scala

```diff
@@ -66,37 +63,6 @@ object MillLspSuite extends BaseImportSuite("mill-import") {
     }
   }
 
-  testAsync("force-command") {
```

Comment:
Those are testing the same things as the sbt tests and it seems Mill tests are less stable.